### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by mikesch803**
1. add the routing

# files to check

- [ ] src/App.js
- [ ] src/components/header/Header.jsx
- [ ] src/pages/home/Home.jsx
- [ ] src/pages/quiz/Quiz.jsx
- [ ] src/pages/result/Result.jsx

### CODE REVIEW

File: src/data/quiz-data.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+export const quiz = [
+  {
+    _id: uuid(),
+    question:
+      "Lorem ipsum dolor, sit amet consectetur adipisicing elit. ullam eveniet, laudantium harum! Quam, eos esse?",
+    option1: { option:"option 1", answer:true },
+    option2: { option:"option 2", answer:false },
+    option3: { option:"option 3", answer:false }
+  },
```
Comment (anik31) [inline src/data/quiz-data.js:11]: suggestion - later you can use APIs for fetching quiz data instead of storing data in array

File: src/index.js [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
-import App from './App';
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+import App from "./App";
+import { BrowserRouter as Router } from "react-router-dom";
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
+const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
```
Comment (anik31) [inline src/index.js:8]: appreciation - good that you're using react-18 syntax to avoid warnings

File: src/pages/quiz/Quiz.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  .main-quiz {
+    grid-area: main;
+    margin-top: 2rem;
+  }
+  
+  .quiz {
+    gap: 1rem;
+    border: 1px solid var(--COLOR-SECONDARY);
+    padding: 2rem;
+    border-radius: var(--BORDER-RADIUS);
```
Comment (anik31) [inline src/pages/quiz/Quiz.css:26]: appreciation - nice that you've a variable for border radius. Using this will make the UI consistent.

**Review by anik31 — APPROVED** [review]
Great work @mikesch803 , the file structure & code is clean.

File: src/data/quiz-data.js (hunk 1 above)
Comment (mikesch803) [reply src/data/quiz-data.js:11]: yes i will make the changes later, when i working on functionality

File: src/pages/home/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            className="img-resp"
+          />
+        </div>
+      </main>
+      <section className="home-section">
+        <h1 className="ft-w-700 txt-center" id="quiz">
+          Categories
+        </h1>
+        <ul className="categories-list">
+          {Array(3)
```
Comment (VLeads) [inline src/pages/home/Home.jsx:28]: As far as I understood it. You are making 3 arrays with the given data right ?

File: src/pages/result/Result.jsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+import "./Result.css";
+import { quiz } from "../../data/quiz-data";
+
+export function Result() {
+  return (
+    <div className="quiz-grid-layout">
+      <main className="main-quiz">
+        <ol className="quiz border-none">
+        <h3>Result</h3>
+          {/* <div className=""> */}
```
Comment (VLeads) [inline src/pages/result/Result.jsx]: you can remove comments

**Review by VLeads — APPROVED** [review]
All looks good. Good to go !

File: src/pages/home/Home.jsx (hunk 4 above)
Comment (mikesch803) [reply src/pages/home/Home.jsx:28]: so i used the dummy data as of now, when i will work on functionality, i will change this

**Pull request merged**